packages/components/src/components/form/Input/index.tsx

```diff
@@ -23,8 +25,8 @@ const StyledInput = styled.input<InputProps>`
     padding: 1px ${props => (props.textIndent ? `${props.textIndent[1] + 16}px` : '16px')} 0
         ${props => (props.textIndent ? `${props.textIndent[0] + 16}px` : '16px')};
     font-size: ${variables.FONT_SIZE.SMALL};
-    border-radius: 4px;
-    border: solid 2px
+    border-radius: ${props => props.borderRadius}px;
+    border: solid ${props => props.borderWidth}px
```

Comment:
this could lead to 
```
border-radius: px;
border: solid px  #e8e8e8; 
```
if the props are not defined...

Reply:
now I see that they have defined default value, but they are marked as possible undefined in InputProps interface...

Reply:
possible undefined removed, thx